bigquery_etl/parse_metadata.py

```diff
@@ -0,0 +1,87 @@
+"""Parsing of metadata yaml files."""
+
+import re
+import yaml
+import os
+
+
+METADATA_FILE = "metadata.yaml"
+
+
+class Metadata:
+    """Representation of metadata file content."""
+
+    def __init__(self, friendly_name=None, description=None, labels={}):
+        """Create a new Metadata instance."""
+        self.friendly_name = friendly_name
+        self.description = description
+        self.labels = labels
+
+    @staticmethod
+    def is_valid_label(label):
+        """
+        Check if a label has the right format.
+
+        Only hyphens (-), underscores (_), lowercase characters, and
+        numbers are allowed. International characters are allowed.
+        """
+        return re.match(r"[\w\d_-]+", label)
+
+    @classmethod
+    def from_file(cls, metadata_file):
+        """Parse metadata from the provided file and create a new Metadata instance."""
+        friendly_name = None
+        description = None
+        labels = {}
+
+        with open(metadata_file, "r") as yaml_stream:
+            try:
+                metadata = yaml.safe_load(yaml_stream)
+
+                if "friendly_name" in metadata:
+                    friendly_name = metadata["friendly_name"]
```

Comment:
```suggestion
                friendly_name = metadata.get("friendly_name", None)
```

This is a more compact notation that avoids repeating the same string literal twice. It also means you can avoid explicitly initializing the variable outside this context.

Can also be applied to description below.